subschemas/substrate-api-proxy/src/typeDefs/bounties.ts

```diff
@@ -1,27 +1,47 @@
 export default /* GraphQL */ `
   type BountiesSummary {
-    activeBounties: Int!
+    activeBounties: String!
```

Comment:
Why string?

Reply:
Technically it is `Number` but for values like this most of the cases we have to use string for `BN`. That's why changed it too.